Sources/PointFree/Routers/PointFreeRoutes.swift

```diff
@@ -49,6 +60,15 @@ private let routers: [Router<Route>] = [
   Route.iso.account
     <¢> get %> lit("account") <% end,
 
+  Route.iso.admin <<< Route.Admin.iso.index
+    <¢> get %> lit("admin") <% end,
+
+  Route.iso.admin <<< Route.Admin.iso.newEpisodeEmail <<< Route.Admin.NewEpisodeEmail.iso.send
+    <¢> post %> lit("admin") %> lit("new-episode-email") %> pathParam((.int) >>> (.tagged)) <% lit("send") <% end,
```

Comment:
I added a `.tagged` combinator so that we can do stuff like this `(.int) >>> (.tagged)` to parse a `Tagged<_, Int>`.

Reply:
Love it.